src/replacer.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function getReplacer() {
```

Comment:
This is not referenced at all in `index.js` nor exported to be manually given in by the user. Seems to work but impossible to use?

Reply:
Silly mistake, but good catch! I've decided that it's unlikely that using the default replacer will have an impact on projects using an older version currently. I've gone ahead and made it use the internal replacer module by default, therefore, no need to expose that module for users.